tests/libcxx/expected_results.txt

```diff
@@ -505,7 +505,7 @@ std/strings/char.traits/char.traits.specializations/char.traits.specializations.
 
 # C++20 P1135R6 "The C++20 Synchronization Library"
 std/atomics/types.pass.cpp FAIL
-std/atomics/atomics.types.operations/atomics.types.operations.wait/atomic_wait.pass.cpp FAIL
+std/atomics/atomics.types.operations/atomics.types.operations.wait/atomic_wait.pass.cpp PASS

```

Comment:
Passing tests should simply be removed from `expected_results.txt`, and also from `skipped_tests.txt` which controls the Microsoft-internal test harness. (The only tests that should appear in this file as `PASS` are those that libcxx would ordinarily expect to fail.)